src/guest/vm_builder_qemu.cc

```diff
@@ -647,7 +647,11 @@ void VmBuilderQemu::BuildAafCfg(void) {
 
         std::string aaf_suspend = cfg_.GetValue(kGroupAaf, kAafSuspend);
         if (!aaf_suspend.empty()) {
-            aaf_cfg_->Set(kAafKeySuspend, aaf_suspend);
+            if (aaf_suspend.compare("true") == 0 || aaf_suspend.compare("false") == 0) {
```

Comment:
I am thinking about compatible with enable/disable.
true/enable  ==> aaf_cfg_.Set(kAafKeySuspend, "true");
false/disable ==> aaf_cfg_.Set(kAafKeySuspend, "false");
others ==> report error and refuse to start VM